assets/js/components/adminbar/AdminBarZeroData.js

```diff
@@ -19,15 +19,19 @@
 /**
  * WordPress dependencies
  */
-import { withFilters } from '@wordpress/components';
-import { Component, Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
 
-class LegacyAdminBarModules extends Component {
-	render() {
-		return (
-			<Fragment />
-		);
-	}
-}
+const AdminBarZeroData = () => {
+	return (
+		<div>
+			<h4 className="googlesitekit-adminbar__title">
```

Comment:
This looks a bit too large IMO - I think it might be a bit cleaner to use the same class here and apply a bold weight. We could use a new `googlesitekit-font-weight-bold` utility for this rather than changing adminbar specific styles.

Current:
![image](https://user-images.githubusercontent.com/1621608/106455971-9ce39f80-6495-11eb-81a5-b78e9061034c.png)

Proposed:
![image](https://user-images.githubusercontent.com/1621608/106455984-a1a85380-6495-11eb-8b19-1de175b5fe0c.png)

Then the two lines don't look so close together either. What do you think?

Also, we probably shouldn't use an actual heading element here (maybe even a `p` either) to avoid potentially inheriting styles from the theme and rely on our classes to provide all of the styling.

Reply:
By " use the same class here " did you mean the subtitle class? I've applied the subtitle class and the bold class for the title as well as replacing the `h4` with a `p`.

Reply:
Also, I couldn't find the `googlesitekit-font-weight-bold` utility: https://github.com/google/site-kit-wp/search?q=googlesitekit-font-weight-bold

Reply:
> By " use the same class here " did you mean the subtitle class? I've applied the subtitle class and the bold class for the title as well as replacing the `h4` with a `p`.

@benbowler Yep, sorry for the ambiguity there, but sounds like you understood what I meant 😄 

> Also, I couldn't find the `googlesitekit-font-weight-bold` utility

I meant we'd need to create that utility for the font weight. We have a handful of different utilities for this kind of thing here: https://github.com/google/site-kit-wp/tree/develop/assets/sass/utilities
Basically one module per css property and one class per rule/value combination. You'll see what I mean with the others there. Apologies if that wasn't clear.